packages/core/utils/lib/content-types.js

```diff
@@ -100,7 +100,7 @@ const getPrivateAttributes = (model = {}) => {
 };
 
 const isPrivateAttribute = (model = {}, attributeName) => {
```

Comment:
```suggestion
const isPrivateAttribute = (model, attributeName) => {
```

Doesn't look like default value is useful here